rmc/pipeline/calculate_missense_badness.py

```diff
@@ -50,7 +54,14 @@ def main(args):
         "This regional missense constraint script calculates missense badness."
     )
     parser.add_argument(
-        "--overwrite", help="Overwrite existing data.", action="store_true"
+        "--overwrite-temp",
+        help="Overwrite existing intermediate temporary data, for use in functions where existing final outputs may be modified.",
```

Comment:
I'm not sure about the wording of the "for use part". How about:
```
Overwrite existing temporary data, for use in functions with option to modify existing final output data.
```
or
```
Overwrite existing temporary data, for use in functions with option to overwrite existing final output data.
```
mostly just changing this to active voice